src/observers/infoMessageObserver.ts

```diff
@@ -8,9 +8,10 @@ export class InfoMessageObserver {
     constructor(private _environmentController: EnvironmentController) { }
 
     public eventHandler = (event: BaseEvent) => {
+        let asUserSignInCompleted;

```

Comment:
why do we need to move the variable definition here?

Reply:
Here is the rule suggesting this https://eslint.org/docs/rules/no-case-declarations

Reply:
OK, then how about wrapping the block since this variable is only used in this case.